lib/datapacksjob.js

```diff
@@ -486,6 +485,13 @@ DataPacksJob.prototype.formatResponse = async function(jobInfo, error) {
                     dataPack.Diffs = dataPack.DiffType != 'Unchanged';
                 }
 
+                if (!dataPack.orgUrl && jobInfo.VlocityDataPackKeyToUrlMapping[dataPack.VlocityDataPackKey]) {
+                    dataPack.orgUrl = jobInfo.VlocityDataPackKeyToUrlMapping[dataPack.VlocityDataPackKey];
+                    if(dataPack.orgUrl){
+                        dataPack.orgUrl = dataPack.orgUrl.replace(/%vlocity_namespace%__/g,this.vlocity.namespacePrefix);
```

Comment:
I don't think you need this replace because it is already in the `getVisualForcePageUrl` function.

Reply:
Though its getting replaced while debug, but in real time run, its coming as "%vlocity_namespace%"

Reply:
Ok, seems good then.